src/scenechange/mod.rs

```diff
@@ -367,7 +372,7 @@ impl<T: Pixel> SceneChangeDetector<T> {
           * (distance_from_keyframe - min_keyint) as f64
           / (max_keyint - min_keyint) as f64
     };
-    let threshold = intra_cost * (1.0 - bias);
+    let threshold = intra_cost * (1.0 - bias) / 2.2;
```

Comment:
You should be able to change THRESH_MAX rather than adding another constant here.